src/ripple/app/main/Application.cpp

```diff
@@ -877,19 +877,21 @@ class ApplicationImp
             if (! config_->standalone())
             {
                 boost::filesystem::space_info space =
-                        boost::filesystem::space (config_->legacy ("database_path"));
+                    boost::filesystem::space (
+                        config_->legacy ("database_path"));
 
-                // VFALCO TODO Give this magic constant a name and move it into a well documented header
-                //
+                // VFALCO TODO Give this magic constant a name and
+                // move it into a well documented header
                 if (space.available < (512 * 1024 * 1024))
                 {
                     JLOG(m_journal.fatal())
                         << "Remaining free disk space is less than 512MB";
                     signalStop ();
                 }
-            }
 
-            m_jobQueue->addJob(jtSWEEP, "sweep", [this] (Job&) { doSweep(); });
+                m_jobQueue->addJob(
+                    jtSWEEP, "sweep", [this] (Job&) { doSweep(); });
```

Comment:
How heavily would you have to be using rippled in standalone for not sweeping to become an issue?
Would it be worth adding a `sweep` command to manual trigger sweeping?